### DISCUSSION TITLE
Jackson Community is huge, maybe we could get a Discord going?

### DISCUSSION THREAD

**Discussion by rikkarth** [FasterXML/jackson-databind#4710] [Q&A] [answered]
I find the jackson community amazing and actually big. Have we considering open our discord?

I feel just in general it would help a lot in communication and also share ideas.

**Comment by JooHyukKim** [comment, marked as answer]
I personally think Github Discussion would suffice.
The community of regularly participate in maintenance is actually small.
Actually there is [discussion on jackson-dev](https://groups.google.com/g/jackson-dev/c/KZvHquyesZI/m/Z_O8UQlpAgAJ) mailing list about completely moving to Github discussion.

**Reply by rikkarth** [reply]
> I personally think Github Discussion would suffice. The community of regularly participate in maintenance is actually small. Actually there is [discussion on jackson-dev](https://groups.google.com/g/jackson-dev/c/KZvHquyesZI/m/Z_O8UQlpAgAJ) mailing list about completely moving to Github discussion.

Ok, that's for sure a good start. I will leave this idea for now, I believe GitHub discussions are for sure an enhancement.

**Reply by JooHyukKim** [reply]
Yup, even better go ahead and share your thoughts on the discussion? :))
